### ISSUE TITLE
Why is centos5/6 docker so different than other redhat containers?

### ISSUE THREAD

**Issue by gdevenyi** [packpack/packpack#29]
Seems like a lot of special-casing going on there.

**Comment by gdevenyi** [comment]
I'd like to enable ccache in centos5, I think I can do so with:
```
ENV CC="ccache /opt/rh/devtoolset-2/root/usr/bin/gcc"
ENV CXX="ccache /opt/rh/devtoolset-2//root/usr/bin/g++"
```

**Comment by rtsisyk** [comment]
> Why is centos5/6 docker so different 

Because CentOS 5/6 are ancient idiotic distros which don't include C99-compatible compiler by default.

**Comment by rtsisyk** [comment]
>  I'd like to enable ccache in centos5, I think I can do so with:
>
>ENV CC="ccache /opt/rh/devtoolset-2/root/usr/bin/gcc"
>ENV CXX="ccache /opt/rh/devtoolset-2//root/usr/bin/g++"

I think it is OK.

**Comment by gdevenyi** [comment]
> Because CentOS 5/6 are ancient idiotic distros which don't include C99-compatible compiler by default.

EW.

Addressed with PR #31

**Comment by rtsisyk** [comment]
> EW

Sorry :)
I pushed your fix. Thanks!
